go.mod

```diff
@@ -48,17 +48,18 @@ require (
 	github.com/lightningnetwork/lnd/ticker v1.0.0
 	github.com/ltcsuite/ltcd v0.0.0-20190101042124-f37f8bf35796
 	github.com/mattn/go-runewidth v0.0.9 // indirect
-	github.com/miekg/dns v0.0.0-20171125082028-79bfde677fa8
-	github.com/prometheus/client_golang v1.0.0
+	github.com/miekg/dns v1.0.14
+	github.com/prometheus/client_golang v1.11.0
 	github.com/stretchr/testify v1.7.0
 	github.com/tv42/zbase32 v0.0.0-20160707012821-501572607d02
 	github.com/urfave/cli v1.20.0
-	go.etcd.io/etcd v3.4.14+incompatible
-	golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
-	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7
-	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2
-	google.golang.org/grpc v1.29.1
-	google.golang.org/protobuf v1.23.0
+	go.etcd.io/etcd/client/pkg/v3 v3.5.0
```

Comment:
nit: I think we need to bump the `kvdb` module version too here (then tag later).

Reply:
Good point. I think we normally did "merge PR -> tag submodule -> create PR to bump submodule". But since merging such a PR will break external users anyway until the tag is pushed we might as well avoid the second PR by bumping it in advance.